src/wallet/validatorKeys.ts

```diff
@@ -2,20 +2,29 @@ import { ErrInvariantFailed } from "../errors";
 import { guardLength } from "./assertions";
 import { parseValidatorKey } from "./pem";
 
-const bls = require("@multiversx/sdk-bls-wasm");
-
 export const VALIDATOR_SECRETKEY_LENGTH = 32;
 export const VALIDATOR_PUBKEY_LENGTH = 96;
 
 export class BLS {
     private static isInitialized: boolean = false;
+    public static bls: any;
```

Comment:
A bit tricky. C#'s `internal` would have been nice here :)

Though, since we plan to drop Herumi's library, it's fine :+1: 

Alternative: private field and a public function `getUnderlyingComponent()` (or something similar).

Reply:
I will let it like this since we plan to drop it